backend/src/main/java/codezap/auth/service/AuthService.java

```diff
@@ -0,0 +1,43 @@
+package codezap.auth.service;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import codezap.auth.dto.LoginAndCredentialDto;
+import codezap.auth.dto.request.LoginRequest;
+import codezap.auth.dto.response.LoginResponse;
+import codezap.auth.provider.CredentialProvider;
+import codezap.global.exception.CodeZapException;
+import codezap.member.domain.Member;
+import codezap.member.repository.MemberRepository;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class AuthService {
+
+    private final CredentialProvider credentialProvider;
+    private final MemberRepository memberRepository;
+
+    public LoginAndCredentialDto login(LoginRequest loginRequest) {
+        Member member = getMemberByIdAndPassword(loginRequest.loginId(), loginRequest.password());
+        String credential = credentialProvider.createCredential(member);
+        return LoginAndCredentialDto.from(LoginResponse.from(member), credential);
+    }
+
+    private Member getMemberByIdAndPassword(String loginId, String password) {
```

Comment:
정말 사소한건데, 지금의 메서드 명이 검색 조건으로 password 를 같이 사용하는 것 처럼 느껴져서요!
조금 바꿔 보았는데, 저도 괜찮은 메서드 명인지 확신은 없네용
한번 가볍게만 읽어봐 주세요!
```suggestion
    private Member getVerifiedMemberById(String loginId, String password) {
```

Reply:
Password도 확인하는 부분이 있어서`getVerifiedMember`로 개선하겠습니당